.github/workflows/test.yml

```diff
@@ -85,8 +85,8 @@ jobs:
       - name: Upload Result
         uses: actions/upload-artifact@v4
         with:
-          compression-level: 0
-          name: SA.CLEO5
+          compression-level: 6
+          name: SA.CLEO_5-${{ env.SHORT_SHA }}
```

Comment:
jobs can return values via outputs. See how we use `version` in Upload Release job

Reply:
we can also read version number from cleo.h to name archive as `SA.CLEO_5.0.1-SHA` for example